packages/design-tokens/bin/buildCssVarTokens.ts

```diff
@@ -70,16 +75,24 @@ const run = () => {
   fs.mkdirSync(jsonOutput, { recursive: true })
   fs.mkdirSync(cssOutput, { recursive: true })
 
-  // Any theme passed into the factory function will be fine, as they all have the same keys
-  const customPropertiesTheme = makeCSSVariableTheme(
-    augmentThemeForSassVariables(defaultTheme)
+  const customPropertiesThemeVersion2 = makeCSSVariableTheme(
```

Comment:
Is there a more descriptive name you could use for these variables? I'm unsure about what they do on first glance.

Reply:
It's a doozie to describe but i've renamed and added comments